spec/models/pawn_spec.rb

```diff
@@ -0,0 +1,120 @@
+require 'rails_helper'
+
+RSpec.describe Pawn, type: :model do
+  before(:all) do
+    User.delete_all
```

Comment:
It's probably out of scope for this PR, but it might be worth looking into https://github.com/DatabaseCleaner/database_cleaner, as that will automatically ensure the DB is clean at the start of each test.